XYZDrivers/src/main/resources/connection.properties

```diff
@@ -0,0 +1,4 @@
+# Connection configuration. **DO NOT** check in changes to this relating to your personal environment, or you'll break it for everyone else.
```

Comment:
This should have been ignored. I think you committed it then updated the .gitignore file. If you run git rm --cached XYZDrivers/src/main/resources/connection.properties that should remove it from git.

Reply:
I checked it in deliberately so people have an example to edit. Subsequent changes shouldn't get committed in though

Reply:
Ok cool! Just need to remember to remove it once everyone has it

Reply:
I don't see how removing it will help, at least if we leave it in you can see what you need to change when in a new environment